app/pages/multisig/AccountTransactions/MultisignatureAccountTransactions.module.scss

```diff
@@ -1,44 +1,2 @@
 @import 'config/bundle';
 @import '../common/MultiSignatureFlowPage.module.scss';
```

Comment:
Does it make sense to keep this file now that it only contains other imports?

Reply:
Not really, I changed the users to use `../common/MultiSignatureFlowPage.module.scss` instead.